Sources/MapboxNavigation/Expression.swift

```diff
@@ -30,4 +36,86 @@ extension Expression {
             }
         }
     }
+    
+    func localized(into locale: Locale?) -> Expression {
+        switch elements.first {
```

Comment:
This usage is now broken as of MapboxMaps v2.0.0-beta.18: mapbox/mapbox-maps-ios#299.